### PR TITLE
TW-543 Improve the 'Buy' section

### PR DESCRIPTION
**Pull request by keshan3262**

### CODE REVIEW

File: src/utils/analytics/use-analytics.hook.ts [hunk 1]
```diff
@@ -12,39 +12,45 @@ export const useAnalytics = () => {
...
-      properties: {
+  const trackEvent = useCallback(
+    async (
+      event?: string,
+      category: AnalyticsEventCategory = AnalyticsEventCategory.General,
+      additionalProperties: AnalyticsEventProperties = {}
+    ) =>
+      event !== undefined &&
+      analyticsEnabled &&
+      jitsu.track(category, {
```
Comment (lourenc) [inline src/utils/analytics/use-analytics.hook.ts:23]: Should the analytics change be in the scope of this PR? Please, let me know :)

**Review by lourenc — CHANGES_REQUESTED** [review]
Really GJ. Minor points highlighted, let me know if you disagree with something.

File: src/utils/analytics/use-analytics.hook.ts (hunk 1 above)
Comment (keshan3262) [reply src/utils/analytics/use-analytics.hook.ts:23]: I think it should be there because it is a part of performance optimization

**Review by lourenc — APPROVED** [review]

**Review by lourenc — APPROVED** [review]

**Review by lourenc — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
build number: 1.14_607 Android done

**Review by vladvn1905 — APPROVED** [review]
build number: 1.14_607 ios done

**Review by lourenc — DISMISSED** [review]

**Review by lourenc — APPROVED** [review]

**Pull request merged**